prowler/providers/azure/services/vm/vm_service.py

```diff
@@ -50,7 +50,7 @@ def _get_virtual_machines(self):
                                     else None
                                 ),
                                 location=vm.location,
-                                security_profile=vm.security_profile,
+                                security_profile=vm.get("security_profile", None),
```

Comment:
Is not `.get` only for dictionaries? I think you should use `getattr`.

Reply:
You're right, fixed.